src/Redux/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+import { combineReducers } from './node_modules/redux';
```

Comment:
🛑the path `./node_modules/redux` does not exist. just do this `import { combineReducers } from 'redux';`